crm_claim_type/README.rst

```diff
@@ -4,7 +4,7 @@
 CRM Claim Types
 ===============
 
-Includes a way to classify claims adding the concept of type.
+Includes a way to classify claims adding the concept of type and with this it allows to configure claim stages depending on claim types.
```

Comment:
@yaniaular what use cases must be included? Would it be just fine with some examples?